front-end/bundle-configuration/default/runtime-config.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+export APPLICATION_SECRET=PPjOW0n2aV?s@6RdiNV@7/5xJhiiKTzk[VdHjkU9YHit8sLHoJ1rp0DCCn6b=lXt
```

Comment:
Alright, you guys know better, but I feel a comment about what this file is for and how it's intended to be used would be useful.